scipy/io/arff/arffread.py

```diff
@@ -475,7 +475,9 @@ def split_data_line(line, dialect=None):
     # Remove the line end if any
     if line[-1] == '\n':
         line = line[:-1]
-
+    
+    line= line.strip()
```

Comment:
Would it be possible to provide a small regression test based on the example in gh-12904. It seems that `pandas` isn't actually required to reproduce the problem.

I'm not sure if we'd want to directly include the data file linked to from that issue or if it would be possible to produce something even smaller to mock the problem, etc.

Reply:
Hi @tylerjereddy !
I'm not clear on how to proceed. I saw this issue as a minor bug and rectified it.